src/services/services.ts

```diff
@@ -1907,6 +1907,58 @@ namespace ts {
             return codefix.getAllFixes({ fixId, sourceFile, program, host, cancellationToken, formatContext });
         }
 
+        function organizeImports(scope: OrganizeImportsScope, formatOptions: FormatCodeSettings): ReadonlyArray<FileTextChanges> {

```

Comment:
I think these changes are substantial enough to deserve an `organizeImports.ts` file, just like we have for breakpoints/classifier/etc.